app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/ui/notification/DeviceShieldAlertNotificationBuilder.kt

```diff
@@ -51,15 +51,6 @@ object DeviceShieldAlertNotificationBuilderModule {
 
 interface DeviceShieldAlertNotificationBuilder {
 
-    fun buildReminderNotification(
```

Comment:
My ultimate goal is to get rid of this class and have each notification have its own VpnStaticNotificationContentPlugin implementation.